geps/gep-1742.md

```diff
@@ -1,7 +1,7 @@
 # GEP-1742: Timeouts
 
 * Issue: [#1742](https://github.com/kubernetes-sigs/gateway-api/issues/1742)
-* Status: Provisional
+* Status: Implementable
```

Comment:
The more I think about it the more I feel like we should merge this PR in a `Provisional` state to allow just a bit more soak time, and to invite others to bring their own PRs to the table prior to when we would start implementation. We could bring the GEP up at the next couple of weekly syncs just to let people know now's a good time to chime in, and maybe add a few commits and ideas of their own.

Let me know your thoughts?